portfolio/src/main/java/com/google/sps/servlets/ServerCommentsServlet.java

```diff
@@ -0,0 +1,59 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.ReviewerData;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/comments")
+public final class ServerCommentsServlet extends HttpServlet {
+
+  private final List<ReviewerData> reviews = new ArrayList<>();
+  /**
+   * @return the request parameter, or the default value if the parameter
+   *         was not specified by the client
+   */
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+    String value = request.getParameter(name);
+    if (value == null) {
+      return defaultValue;
+    }
+    return value;
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get & store the input from the comment section.
+    String reviewerName = getParameter(request, "name", "Anonymous");
+    String reviewerComment = getParameter(request, "comment", "");
+    reviews.add(new ReviewerData(reviewerName, reviewerComment));
```

Comment:
Consider in the next iteration to validate user input.

Reply:
Should I still do that if the JavaScript puts all the input in a bulletproof text box (that starts and ends with quote marks ") or double checking is always good (because I can't always know where the input will be used, especially when the app scales)?